### DISCUSSION TITLE
Card preview in Kanban View (Projects Beta)

### DISCUSSION THREAD

**Discussion by martinsangolt** [community/community#5884] [Projects and Issues]
It would be great if it was possible to view and edit the information in the different issues, without needing to open a dedicated url — see both the kanban board and content of one of the selected issues, in a slit view.

**Comment by nickhagen** [comment]
existing non-beta projects open issue in a slide out, which in my opinion is perfect! with a button to open the "full issue" if needed

**Reply by alindst** [reply]
A sidebar view like the Project Board feature helps with quick standups. I know it seems silly complaining about waiting 1-2 secs for a URL to open in a  new tab but it adds a lot of friction switching between tabs/windows when you just want a quick glance of issue details. This one feature difference has made be go back to project boards several times over beta.

**Reply by ssnowden** [reply]
Would wholeheartedly agree with this. I like to use old-style cards to keep a next steps checklist open in 'In Progress' to keep track of my thinking between workdays. Reduces cognitive load thinking about - what am I doing today. Especially useful when working on multiple projects in a week.

**Comment by steebchen** [comment]
This is related: https://github.com/github/feedback/discussions/5381

**Comment by steebchen** [comment]
This is now in alpha as a feature preview 
<img width="879" alt="image" src="https://user-images.githubusercontent.com/5013932/169647601-e6a41ad8-5c9f-4b47-8a76-9a286357567c.png">

**Comment by stilliard** [comment]
This request is now implemented ref https://github.com/orgs/community/discussions/5381 , super useful for issues, hopefully Pull Requests can have this soon as well.
